src/components/widget/Lookup/Lookup.js

```diff
@@ -401,17 +369,18 @@ class Lookup extends Component {
             // if (index < 2 && this.props.properties[0].field === "C_BPartner_ID") {
             // TODO: This is really not how we should be doing this. Backend should send
             // us info which fields are usable with barcode scanner
-            showBarcodeScannerBtn = item.field === 'M_LocatorTo_ID';
+            const field = item.field;
+            showBarcodeScannerBtn = field === 'M_LocatorTo_ID';
```

Comment:
Why is this hardcoded ?

Reply:
Cause we don't have a flag on the backend side. This is a `special` field that tells us if we should render barcode scanner on not. It's literally described in the comment above.
Also not a part of this PR so not relevant here.